pymodbus/client/mixin.py

```diff
@@ -502,7 +502,7 @@ def read_fifo_queue(self, address: int = 0x0000, **kwargs: Any) -> ModbusRespons
     # code 0x2B sub 0x0D: CANopen General Reference Request and Response, NOT IMPLEMENTED
 
     def read_device_information(
-        self, read_code: int = None, object_id: int = 0x00, **kwargs: Any
+        self, read_code: Union[int, None] = None, object_id: int = 0x00, **kwargs: Any
```

Comment:
Union ?? I think we should either use "Union" or "|" not both.

Reply:
Use `from __future__ import annotations` to allow `|` to be used.

I will mark the other redundant comments Resolved.